src/main/common/components/ccj/paid-amount-summary.njk

```diff
@@ -20,13 +19,22 @@
         </tr>
         </thead>
         <tbody>
-        <tr>
-          <th scope="row">{{ t('Claim amount') }}</th>
-          <td class="numeric last">
-            <span class="font-small">{{ claim.claimData.amount.totalAmount() | numeral }}</span>
-          </td>
-        </tr>
-        {% if interestDetails %}
+        {% if not amountSettledFor %}
+          <tr>
+            <th scope="row">{{ t('Claim amount') }}</th>
+            <td class="numeric last">
+              <span class="font-small">{{ claim.claimData.amount.totalAmount() | numeral }}</span>
+            </td>
+          </tr>
+        {% else %}
+          <tr>
+            <th scope="row">{{ t('Amount you are settling for') }}</th>
```

Comment:
I'm pretty sure this should say "Amount you're settling for" but struggling to find the prototype page for this content

Reply:
There is no prototype, this content was written by Chris and signed off by marc